### PR TITLE
Add tokens on Wanchain mainnet

### PR DESCRIPTION
**Pull request by ssc806**
Add this files for supporting the Wanchain tokens transaction via Trezor Device. Wanchain is using the same chain ID as Ethereum, so these files should merge to the eth directory.

### CODE REVIEW

File: tokens/eth/0xA13a41a2349D09C1E3a0dD4d9448Af4aD799fC4A.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "MDX",
+  "name": "BlockMedx",
+  "type": "ERC20",
+  "address": "0xA13a41a2349D09C1E3a0dD4d9448Af4aD799fC4A",
```
Comment (ligi) [inline tokens/eth/0xA13a41a2349D09C1E3a0dD4d9448Af4aD799fC4A.json]: this ERC-55 checksum is wrong

**Comment by ligi** [comment]
0x51316966907EF568689775066E97C46A4fFDE1A4 does not look like a contract - can you explain?

**Comment by ssc806** [comment]
> 0x51316966907EF568689775066E97C46A4fFDE1A4 does not look like a contract - can you explain?

These contracts are deployed on Wanchain Mainnet, e.g. https://www.wanscan.org/token/0x51316966907ef568689775066e97c46a4ffde1a4 is for the one you specified.

Wanchain's code is forked from Ethereum and using the same chainID as Ethereum. For the Trezor Firmware, Wanchain and Ethereum are sharing the same app and using the same code logistics. They are using the tx_type to distinguish each other.

So, to make the Trezor device work with the Wanchain tokens, I need to push the Wanchain's token files here.

**Comment by ligi** [comment]
who had that glorious idea to use the same chainID. Really not sure I want to merge something like this ..

**Pull request closed without merging**